client/src/styles.scss

```diff
@@ -104,10 +133,10 @@ body {
 }
 
 .os-card {
-    max-width: 90%;
-    margin-top: 10px;
-    margin-left: auto;
-    margin-right: auto;
+    max-width: 1200px;
```

Comment:
Accorind to the design team, that was 700, since it should be in the style of [google forms](https://docs.google.com/forms/d/1lXBDqTdN-v6nHNvBRzfORzwxx7FSKGvei7cLnwbGmng/edit)